File: src/main/java/it/pittysoft/affetti/model/DomandaModel.java
package it.pittysoft.affetti.model;


import java.util.Date;

import it.pittysoft.affetti.entity.Domande;
import lombok.Data;

@Data
public class DomandaModel {

	private Date dataProtocollo;
	private String numeroProtocolloDomanda;
	private String tipologia;
	private String stato;
	private String nomeContraente;
	private String cognomeContraente;
	private String Assegnatario;
	private String nomeAss;
	private String cognomeAss;
	private String numeroProtocolloContratto;
	private String comuneDiNascita;
	private String provinciaDiNascita;
	private String statoDiNascita;
	private String comuneDiResidenza;
	private String provinciaDiResidenza;
	private String viaDiResidenza;
	private String civicoDiResidenza;
	private String capDiResidenza;
	private String codiceFiscale;
	private String telefono;
	private String Email;
	private String note;
	private String loculo;
	private String fornice;
	private String comuneDecesso;
	private Date dataDecesso;
	
}

File: src/main/ui/src/app/app-state/models/domanda.model.ts
export class Domanda {

  idDomanda: number;
  idPosto: number;
  idAssegnatario : number;
  idContraente: number;
  numeroProtocolloDomanda: string;
  dataProtocollo: string;
  stato: string;
  tipologia: string;
  nomeContraente: string;
  cognomeContraente: string;
  nomeAss: string;
  cognomeAss: string;
  comuneDiNascita: String;
	provinciaDiNascita: String;
	statoDiNascita: String;
	comuneDiResidenza: String;
	provinciaDiResidenza: String;
	viaDiResidenza: String;
	civicoDiResidenza: String;
	capDiResidenza: Number;
  codiceFiscale: string;
  email: string;
  note: string;
  telefono: Number;
  loculo: string;
  fornice: string;
  data: Date;
  dataDecesso: Date;
  comuneDecesso: string;
  fk_posto: number;
  fk_assegnatario : number;
}

